WordPress/Classes/ViewRelated/Site Creation/Web Address/DesignSystem/Gallery/DSButtonGallery.swift

```diff
@@ -0,0 +1,63 @@
+                DSButton(title: "Tertiary", style: .init(emphasis: .tertiary, size: .large)) { () }
+                DSButton(title: "Primary Disabled", style: .init(emphasis: .primary, size: .large)) { () }
+                    .disabled(true)
+                DSButton(title: "Secondary Disabled", style: .init(emphasis: .secondary, size: .large)) { () }
+                    .disabled(true)
+                DSButton(title: "Tertiary Disabled", style: .init(emphasis: .tertiary, size: .large)) { () }
+                    .disabled(true)
+            }
+            .listRowSeparator(.hidden)
+            .listRowBackground(Color.clear)
+            .listRowInsets(.init(top: Length.Padding.split, leading: 0, bottom: Length.Padding.split, trailing: 0))
+
+            Section("Medium") {
+                HStack {
+                    Spacer()
+                    VStack(alignment: .center, spacing: Length.Padding.medium) {
+                        DSButton(title: "Primary", style: .init(emphasis: .primary, size: .medium)) { () }
+                        DSButton(title: "Secondary", style: .init(emphasis: .secondary, size: .medium)) { () }
+                        DSButton(title: "Tertiary", style: .init(emphasis: .tertiary, size: .medium)) { () }
+                        DSButton(title: "Primary Disabled", style: .init(emphasis: .primary, size: .medium)) { () }
+                            .disabled(true)
+                        DSButton(title: "Secondary Disabled", style: .init(emphasis: .secondary, size: .medium)) { () }
+                            .disabled(true)
+                        DSButton(title: "Tertiary Disabled", style: .init(emphasis: .tertiary, size: .medium)) { () }
+                            .disabled(true)
+                    }
+                    Spacer()
+                }
+            }
+            .listRowSeparator(.hidden)
+            .listRowBackground(Color.clear)
+
+            Section("Small") {
+                HStack {
+                    Spacer()
+                    VStack(alignment: .center, spacing: Length.Padding.medium) {
+                        DSButton(title: "Primary", style: .init(emphasis: .primary, size: .small)) { () }
+                        DSButton(title: "Secondary", style: .init(emphasis: .secondary, size: .small)) { () }
+                        DSButton(title: "Tertiary", style: .init(emphasis: .tertiary, size: .small)) { () }
+                        DSButton(title: "Primary Disabled", style: .init(emphasis: .primary, size: .small)) { () }
+                            .disabled(true)
+                        DSButton(title: "Secondary Disabled", style: .init(emphasis: .secondary, size: .small)) { () }
+                            .disabled(true)
+                        DSButton(title: "Tertiary Disabled", style: .init(emphasis: .tertiary, size: .small)) { () }
+                            .disabled(true)
```

Comment:
I wonder if there's a way to generate this list at runtime so that we can automate adding and removing new elements and also ensure the name is always in sync with the configuration.

```swift
// pseudo-code
let action: String -> Void = { presentAlert(message: "Alert for button with \($0)") }

Emphasis.forEach { emphasis in
  DSButton(title: emphasis.name, style: .init(emphasis: emphasis, size: .small)) { action(emphasis.name) }
}
```

Reply:
I can try to come up with something. However the list shouldn't get modified too often and much longer than this. And by shouldn't I mean that it is not a good idea in terms of keeping a uniform, consistent design in the app.

That being said, it could come handy if we add more components designed the same way.

Reply:
@mokagio hey, I've simplified the boilerplate here a bit. We would still need to provide some strings for the buttons and section but the code looks simpler at least. Wdyt?